### PR TITLE
Add sick68 keyboard

### PR DESCRIPTION
**Pull request by umbynos**
## Add sick68 keyboard

I've added the [sick68](https://www.thingiverse.com/thing:3478494) keyboard. It's similar to the TADA68 layout but it's handwired. The only difference from the original project linked above it's that I used a Pro Micro instead of a teensy 2.0.
Sorry if I did something wrong, It's my first PR to this project, so please be kind :smile: 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/sick68/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 umbynos
```
Comment (fauxpark) [inline keyboards/handwired/sick68/keymaps/default/config.h]: This file can be deleted.

File: keyboards/handwired/sick68/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (fauxpark) [inline keyboards/handwired/sick68/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/handwired/sick68/rules.mk [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (fauxpark) [inline keyboards/handwired/sick68/rules.mk]:
```suggestion
```

File: keyboards/handwired/sick68/sick68.c [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+    return process_record_user(keycode, record);
+}
+
+bool led_update_kb(led_t led_state) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+    return led_update_user(led_state);
+}
+*/
```
Comment (fauxpark) [inline keyboards/handwired/sick68/sick68.c]:
```suggestion
```

File: keyboards/handwired/sick68/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# sick68
```
Comment (fauxpark) [inline keyboards/handwired/sick68/readme.md]:
```suggestion
# SiCK-68
```

File: keyboards/handwired/sick68/config.h [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
```
Comment (fauxpark) [inline keyboards/handwired/sick68/config.h]:
```suggestion
#define PRODUCT_ID 0x5F00
```
```
echo -n "SiCK-68"|shasum
```

File: keyboards/handwired/sick68/config.h [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS \
+    { D3, D2, D1, D0, D4 }
+#define MATRIX_COL_PINS \
+    { C6, D7, E6, B4, B5, B0, D5, B6, B2, B3, B1, F7, F6, F5, F4 }
```
Comment (fauxpark) [inline keyboards/handwired/sick68/config.h]:
```suggestion
#define MATRIX_ROW_PINS { D3, D2, D1, D0, D4 }
#define MATRIX_COL_PINS { C6, D7, E6, B4, B5, B0, D5, B6, B2, B3, B1, F7, F6, F5, F4 }
```

File: keyboards/handwired/sick68/keymaps/default/config.h (hunk 1 above)
Comment (umbynos) [reply keyboards/handwired/sick68/keymaps/default/config.h]: I've used `./util/new_keyboard.sh` utility to generate all the required files, so this file was auto-generated. If It's not useful I'll remove it.

File: keyboards/handwired/sick68/config.h (hunk 7 above)
Comment (drashna) [reply keyboards/handwired/sick68/config.h]: @fauxpark if you use `qmk cformat` or clang-format at all, it does this.

File: keyboards/handwired/sick68/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+    _BASE,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL
+};
+
```
Comment (noroadsleft) [inline keyboards/handwired/sick68/keymaps/default/keymap.c]: You're not using these macro keycodes.

```suggestion
```

File: keyboards/handwired/sick68/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+            if (record->event.pressed) {
+                // when keycode QMKURL is pressed
+                SEND_STRING("https://qmk.fm/\n");
+            } else {
+                // when keycode QMKURL is released
+            }
+            break;
+    }
+    return true;
+}
```
Comment (noroadsleft) [inline keyboards/handwired/sick68/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/handwired/sick68/rules.mk [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (noroadsleft) [inline keyboards/handwired/sick68/rules.mk]:
```suggestion
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs

LAYOUTS = 65_ansi
```

**Comment by noroadsleft** [comment]
Thanks!

For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

**Pull request merged**